src/bookshelf/rebar.lock

```diff
@@ -63,7 +65,7 @@
   0},
  {<<"mini_s3">>,
   {git,"https://github.com/chef/mini_s3",
-       {ref,"edb4a35489ac52b06913b786f5eaede3013abf69"}},
+       {branch,"lbaker/path"}},
```

Comment:
This gets cleaned up once mini_s3 branch merges?

Reply:
Yup, sure does :)  Just temporary cruft.